routes/api/index.js

```diff
@@ -0,0 +1,9 @@
+const router = require("express").Router();
+const usersAndTasksRoutes = require("./usersAndTasks");
+// const tasksRoutes = require("./tasks");
+
+// API routes
+router.use("/api/users", usersAndTasksRoutes);
```

Comment:
should be `router.use("/users", users);`

Reply:
OK this makes sense, I overlooked the extra `/api` path.